tests/lax_numpy_operators_test.py

```diff
@@ -731,6 +731,26 @@ def testSpacingSubnormals(self, dtype):
     self._CheckAgainstNumpy(np.spacing, jnp.spacing, args_maker, check_dtypes=True, tol=0)
     self._CompileAndCheck(jnp.spacing, args_maker, tol=0)
 
+  @jtu.ignore_warning(category=RuntimeWarning, message="divide by zero.*")
+  @jtu.ignore_warning(category=RuntimeWarning, message="invalid value.*")
+  def testFloorDivideByZeroInt(self):
+    x1 = jnp.eye(3, dtype=int)
+    x2 = jnp.zeros((3, 3), dtype=int)
+    jnp_result = jnp.floor_divide(x1, x2)
+    np_result = np.floor_divide(x1, x2)
+    self.assertAllClose(jnp_result, np_result, check_dtypes=True)
+
+  @jtu.sample_product(
+      x1 = [-np.nan, -np.inf, -2., -1., -0., 0., 1., 2., np.inf, np.nan],
+      x2 = [-np.nan, -np.inf, -2., -1., -0., 0., 1., 2., np.inf, np.nan],
```

Comment:
The largest number of cases we regularly test is 90, this sampling is 100 cases, meaning that not all will be tested in practice.

Instead I'd recommend a single test where you define `x = jnp.array([-np.nan, -np.inf...])` and then check the output of `jnp.floor_divide(x[:, None], x[None, :])`

Reply:
Thank! Done.